fhir-model/src/main/java/com/ibm/watson/health/fhir/model/annotation/Binding.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.watson.health.fhir.model.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.FIELD, ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Binding {
+    String bindingName() default "";
+    String strength();
```

Comment:
maybe use BindingStrength.ValueSet instead of String for this?

Reply:
Done.